demo/quiz.css

```diff
@@ -0,0 +1,23 @@
+/* Component styling */
+
+.rq-Question-answerList {
+  list-style: none;
+  margin: 20px 0;
+}
+
+.rq-Answer {
+  margin: 0;
+  padding: 0;
+}
+
+.rq-Answer-button:focus {
+  background-color: #333;
+}
+
+.rq-Answer-button--active {
+  background-color: #333;
+}
+
```

Comment:
just a quick reminder that we don't use an uppercase on class names. It should be ```.rq-answer ```, ```.rq-answer-button:focus``` and ```.rq-answer-button--active```.